src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java

```diff
@@ -93,6 +93,7 @@ public void checkForUpdates() {
                 if (current.equals(getCurrentUrl())) {
                     return true;
                 }
+                return true;
```

Comment:
Why the change here.
THe way the plugin manager checking for updates works is that it will load a different URL `$JENKINS_URL/manage/pluginManager/checkUpdatesServer` this page is an intermediate page - it actually has no content but is a HTTP 302 with zero length content.
This is enough for this funtion now to return true - element is stale - but the page is not back at the updates page and this can cause some funky timing issues.

Reply:
https://github.com/jenkinsci/acceptance-test-harness/pull/1289